pkg/extensions/config/sync/config.go

```diff
@@ -33,6 +33,7 @@ type RegistryConfig struct {
 	RetryDelay       *time.Duration
 	OnlySigned       *bool
 	CredentialHelper string
+	PreserveDigest   bool // sync without converting
```

Comment:
So we are not getting the "stream"-ing support in this PR?

Reply:
^ That is right.

No stream in this PR.